src/WarcraftLegacies.Source/Quests/Frostwolf/QuestMammoth.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MacroTools.QuestSystem;
+using static War3Api.Common;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.LegendSystem;
+
+namespace WarcraftLegacies.Source.Quests.Frostwolf
+{
+  /// <summary>
+  /// Bring any hero to the Echo Isles to unlock the base.
+  /// </summary>
+  public sealed class QuestMammoth : QuestData
+  {
+    private readonly LegendaryHero _rexxar;
+
+    /// <inheritdoc />
+    public QuestMammoth(LegendaryHero rexxar) : base("Lone Wanderer",
+      "Rexxar is always looking to discover new wilds and new creatures, the Borean Tundra offer a perfect ground to explore.",
```

Comment:
I'm not sure if it's really true that Rexxar goes out of his way to search for new creatures, so I think we should just focus on his role as a wanderer.
```suggestion
      "Rexxar's wanderlust has brought him into contact with all kinds of beasts. Yet there is one major landmass he has never ventured to: the cold expanse of Northrend. Surely the wild things roam free even there.",
```